src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_rate.scss

```diff
@@ -172,6 +172,7 @@ The whole widget may have the `is-disabled` state.
     &.is-active {
         color: $color-text-inverted;
         text-decoration: none;
+        font-weight: $font-weight-extrovert;
```

Comment:
This made the button's size change on hover, which was really annoying. After talking to our designer I reverted it in 7c511ba5976642c26b8c0fc31c25a28f3a70ce34.